contracts/erc1155/MintableERC1155.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol";
+
+contract MintableERC1155 is ERC1155PresetMinterPauser {
+    mapping(uint256 => string) public _tokenURIs;
+
+    event Airdrop(address caller, uint256 tokenId, address[] recipients, uint256[] amounts);
+
+    constructor() ERC1155PresetMinterPauser("") {}
+
+    /**
+     * @dev Creates `amounts[i]` new tokens for `recipients[i]` of toke type `tokenId`.
+     */
+    function airdrop(
+        uint256 tokenId,
+        address[] memory recipients,
+        uint256[] memory amounts
+    ) public {
+        require(recipients.length == amounts.length, "MintableERC1155: recipients and amounts length mismatch");
```

Comment:
note that if `recipients` is too long, this function could run out of gas. This is not something we need to catch in the contract but should be a warning for the caller